### PR TITLE
ipywidgets 8.0.3, tests, maintainer

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Notes:
- adds pytest
- adds self as maintainer

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,35 +1,41 @@
...
   host:
     - python >=3.7
     - pip
   run:
-    - python >=3.7
-    - ipython >=4.0.0
     - ipykernel >=4.5.1
-    - traitlets >=4.3.1
+    - ipython >=6.1.0
+    - jupyterlab_widgets >=3.0.4,<4.0.0
```
Comment (bollwyvl) [inline recipe/meta.yaml:23]: i observed some icky behavior with the latest ipywidgets and an even slightly older labextension :crying_cat_face:

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by bollwyvl** [comment]
blocked by https://github.com/conda-forge/jupyterlab_widgets-feedstock/pull/18 (didn't have time to dig in further, seemed the quickest route to getting something less-broken out)

**Review by martinRenou — APPROVED** [review]
Thanks!

**Comment by bollwyvl** [comment]
@conda-forge/ipywidgets  ready for review

**Review by martinRenou — APPROVED** [review]
Thanks!

File: recipe/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipe/meta.yaml:23]: indeed, this fixed the ick on https://github.com/jupyterlite/jupyterlite/pull/892

**Pull request merged**